fontc/src/workload.rs

```diff
@@ -211,35 +281,49 @@ impl<'a> Workload<'a> {
         }
     }
 
-    fn can_run_scan(&self, job: &Job) -> bool {
-        // Job can only run if *no* incomplete jobs exist whose output it could read
-        !self
-            .jobs_pending
-            .iter()
-            .filter(|(other_id, _)| !self.success.contains(other_id))
-            .any(|(_, other_job)| self.might_read(job, other_job))
+    fn can_run_all_or_one(&self, all_or_one: &AllOrOne<AnyWorkId>) -> bool {
```

Comment:
a little comment in plain English similar to the one in the previous `car_run_scan` would help the reader understand what this quite dense `can_run_all_or_one` is doing

Reply:
Oh good catch, added such a comment including explaining why the counter is preferred